lib/identity/client.js

```diff
@@ -366,6 +365,14 @@ class Client extends PartialClient {
     let sharedRecordList = []
     const lengthGroup = groupList.groups.length
     for (var index = lengthGroup - 1; index >= 0; index--) {
+      var groupNameSplit = groupList.groups[index].groupName.split('.')
```

Comment:
extract this bit of validation logic out to a helper function `isValidToznySecretNamespace`